lab_1_classify_by_unigrams/main.py

```diff
@@ -168,6 +297,19 @@ def detect_language_advanced(
     In case of corrupt input arguments, None is returned
     """
 
+    if not isinstance(unknown_profile, dict) or not isinstance(known_profiles, list):
+        return None
+
+    distances = []
+    for profile in known_profiles:
+        profile_name: str = profile['name']
+        distance = compare_profiles(unknown_profile, profile)
+        if distance is None:
```

Comment:
you can replace this by `if not distance`

Reply:
or not?

Reply:
done